documentation/src/docs/getting-started.md

```diff
@@ -0,0 +1,103 @@
+
+```
+./gradlew clean install
+```
+
+### Customizing the Build
+
+#### Building against a custom Flink version
+
+We can check and change the Flink version that Pravega builds against via the `flinkVersion` variable in the `gradle.properties` file.
+
+**Note**: Only Flink versions that are compatible with the latest connector code can be chosen.
+
+#### Building against another Scala version
+
+This section is only relevant if you use [Scala](https://www.scala-lang.org/) in the stream processing application in with Flink and Pravega.
+
+Parts of the Apache Flink use the language or depend on libraries written in Scala. Because Scala is **not** strictly compatible across versions, there exist different versions of Flink compiled for different Scala versions.
+If we use Scala code in the same application where we use the Apache Flink or the Flink connectors, we typically have to make sure we use a version of Flink that uses the same Scala version as our application.
+
+By default, the dependencies point to Flink for Scala **2.11**.
+To depend on released Flink artifacts for a different Scala version, you need to edit the `build.gradle` file and change all entries for the Flink dependencies to have a different Scala version suffix. For example, `flink-streaming-java_2.11` would be replaced by `flink-streaming-java_2.12` for Scala **2.12**.
+
+In order to build a new version of Flink for a different Scala version, please refer to the [Flink documentation](https://ci.apache.org/projects/flink/flink-docs-stable/start/building.html#scala-versions).
+
+## Setting up your IDE
+
+Connector project uses [Project Lombok](https://projectlombok.org/), so we should ensure that we have our IDE setup with the required plugins. (**IntelliJ is recommended**).
+
+To import the source into IntelliJ:
+
+1. Import the project directory into IntelliJ IDE. It will automatically detect the gradle project and import things correctly.
+2. Enable `Annotation Processing` by going to `Build, Execution, Deployment` -> `Compiler` > `Annotation Processors` and checking `Enable annotation processing`.
+3. Install the `Lombok Plugin`. This can be found in `Preferences` -> `Plugins`. Restart your IDE.
+4. Connectors project compiles properly after applying the above steps.
+
+For eclipse, we can generate eclipse project files by running `./gradlew eclipse`.
+
+## Releases
+
+The latest releases can be found on the [Github Release](https://github.com/pravega/flink-connectors/releases) project page.
+
+## Support
+
+Don’t hesitate to ask! Contact the developers and community on the  [Slack](https://pravega-io.slack.com/) if you need any help.
+Open an issue if you found a bug on [Github
+Issues](https://github.com/pravega/flink-connectors/issues).
+
+## Documentation
+See the [Project Wiki](https://github.com/pravega/flink-connectors/wiki) for documentation on how to build and use the Flink Connector library.

```

Comment:
Are we moving this wiki into this code repository? Why reference back then?

Reply:
fixed